### PR TITLE
ignore maven target files; add aquilon primer; provide skeleton for man page inclusion

### PR DESCRIPTION
**Pull request by loomis**
The update to the .gitignore file ignores emacs backup files (for my sanity) and ignores the generated maven output files in the `target` subdirectory. 

The draft aquilon book now contains the 'Aquilon Primer' contents provided by Nick.  The overall text and skeleton will need further work, but this is a good start on the introduction.

The last commit shows the changes that will be necessary to integrate the man pages with the aquilon book.  These are minor: remove docbook declaration at the start and include the dobook namespaces on the first XML element.  I can make these changes; however, I don't want to have two sets of man pages in parallel.  So the question is:

Does MS want to continue to use the `make` build of the man pages?
- If not, it is straightforward to convert everything to the new version. 
- If so, can MS verify that that namespace/declaration change doesn't break the build via `make`?

### CODE REVIEW

**Comment by ned21** [comment]
@njwilliams - did we decide we were OK with the CC licence listed in the pom file or should it be changed to the apache one?

@loomis - having an aq-docs and a doc directory seems confusing. Can we amalgamate them? We can drop the Makefile although it would be nice to keep it if it works. What's the easiest way for me to verify it still works? `cd doc; make` worked fine but I am not sure if that was effected by your change.

**Comment by loomis** [comment]
@ned21, @njwilliams - The CC license was a copy/paste mistake; I've corrected it.

**Comment by loomis** [comment]
@ned21 - The two doc areas was not intended to be a final solution.  I'll work on moving everything to the maven build and then we can check how to get the Makefile working again.

File: aq-docs/src/docbkx/aq-book.xml [hunk 1]
```diff
@@ -109,6 +109,715 @@
...
+      <section>
+        <title>Monitoring</title>
+
+        <para>The Linux build of Aquilon provides monitoring by way of the
+        Lemon system. Lemon configuration is fully integrated with Quattor
+        which means that the specific pieces to monitor are defined by
+        template and personality. Lemon’s advantage over perfdata is the
+        facility for aggregation of metrics to present a view of utilization
+        over an entire grid. Lemon does not provide as much detailed
+        information (at the host level) as PerfData. The Lemon agents that run
```
Comment (ned21) [inline aq-docs/src/docbkx/aq-book.xml]: Please can we remove references to perfdata and AFS.

File: aq-docs/src/docbkx/aq-book.xml [hunk 2]
```diff
@@ -109,6 +109,715 @@
...
+
+          <listitem>
+            <para>A modified template library that includes the standard QWG
+            functionality and adds functionality tailored to Morgan Stanley
+            use-cases.</para>
+          </listitem>
+        </itemizedlist>
+      </section>
+
+      <section>
```
Comment (ned21) [inline aq-docs/src/docbkx/aq-book.xml]: This section on DNS, NIS, NFS, is all very MS specific. I think it should be removed.

**Comment by ned21** [comment]
The doc built fine, although it was something of an assumption on my part to say "mvn package" to initiate the build. That should probably be in a README or something. The example man page mostly worked except for:
- in the man output, the author is listed as "Jane Quattor docs@quattor.org" and asserts "Copyright © 2013 Quattor Collaboration". I think this needs to say "Contributor" rather than any name since each file can have multiple copyright holders. @njwilliams to confirm?
- The HTML version of the book seems to be missing the nice CSS used by the PAN.
- Not a problem with this pull request but the previous changes to aqd.conf.defaults prevents this branch from cleanly merge into upstream. 
- `cp src/docbkx/appendix/commands/add_alias.xml ../doc/commands/add_alias.xml && cd ../doc && make` -- I have confirmed that this version of the xml without the namespace builds cleanly using make.

File: aq-docs/src/docbkx/aq-book.xml (hunk 2 above)
Comment (loomis) [inline aq-docs/src/docbkx/aq-book.xml]: I've removed the last bullet item that discusses MS.  I've also removed the reference to Opal in a previous bullet.  The complete section on naming has been removed.

**Comment by loomis** [comment]
@ned21 I've copied the css for the html into the results.  Removed references to MS specific text.  Put everything in the `doc` subdirectory and made changes to all man pages to ensure that they can build within a single document.  The information about the author(s) are placeholders.  Anything can be used that you think is appropriate.

**Comment by loomis** [comment]
@ned21 The makefile build is certainly broken now because the source files have been moved to be consistent with a maven layout.  This should be checked/modified again if the Makefile will be maintained.

**Comment by ned21** [comment]
Unfortunately commit bb7a0ca53d5ff26d88bb49864b9c738440a832c9 has a lot of conflicts with the version that Gabor just published to sourceforge and I can't get it to merge cleanly. I guess you did the move from doc/commands -> to doc/src/docbkx/appendix/ using a script that fixed up all the include/namespaces -- would it be possible for you to rerun that against the latest branches that Gabor pushed? Everything up to bb7a0ca53d5ff26d88bb49864b9c738440a832c9 rebases cleanly onto that so maybe roll back to that point? Sorry for the pain.

**Comment by loomis** [comment]
@ned21 I'll work on getting a new version that merges cleanly.

**Comment by piojo-zz** [comment]
It seems some sites find the book very useful. Given that MS doesn't like the idea of reshuffling files around for Maven, what would be the preferred way to go?
1. Build the book with Maven, and the rest of Aquilon with a `setup.py`.
2. Drop the Maven stuff and build the book from a script that will get called by the `setup.py`.

I prefer 1.

**Comment by jrha** [comment]
I prefer 1 and yes, the book is very very useful.

**Comment by piojo-zz** [comment]
So, I have tried and I have something that I can merge. But it fails to build the docbook, and I'm not digging into the XMLs to see what is wrong where.

@loomis, do you have a script that moves xmls from the `doc` directory to `doc/src/docbx` ?

**Pull request closed without merging**